MyCryptoHQ/MyCrypto
# Hide equivalent values for RSK network

Closes #2091

### Description

RSK currency symbol is SBTC "Smart Bitcoin" that crashes to SBTC "Super Bitcoin". This generates mistakes in rate conversions because MyCrypto API returns "Super Bitcoin" rate for SBTC.

### Changes

* Addition of new configuration parameter `hideEquivalentValues` (optional boolean) that avoids showing rate conversions and, consequently, misunderstandings operating with RSK network.

### Steps to Test

1. Select RSK network
2. Go to "View&Send" / "Contracts" tab and prepare a transaction.
3. Check that no USD values are shown.
4. Check that "Equivalent Values" box shows a message about not showing values for this unit.
